README.md

```diff
@@ -39,11 +40,37 @@ brew install git-ignore
 brew uninstall git-ignore</pre></p>
 </details>
 
+### Build with cargo
+
+> NOTE: 
+> 
+> Tested on Archlinux
+
+Download Rust and Cargo with your own distribution/operating system. Clone the repository with git: 
+```bash
+git clone https://github.com/janniks/git-ignore.git
+```
+
+cd into directory: 
+```bash
+cd git-ignore
+```
+
+Launch `cargo build` command: 
+```bash
+cargo build
+```
+
+The binary is generated into `target/debug` directory, if you want to execute as command in any place of your system, you have to add in a directory covered by PATH environment variable. I.e. on linux systems you can copy into `/usr/bin` directory: 
+```bash
+sudo mv target/debug/git-ignore /usr/bin/git-ignore
+```
+
 ## Usage 🚀
 
 **git-ignore** is used to generate new `.gitignore` files or append content to existing ones.
 
-From now on, you simply run `git ignore` to launch the interactive CLI, anytime you want to setup or change a `.gitignore` file.
+From now on, you simply run `git-ignore` to launch the interactive CLI, anytime you want to setup or change a `.gitignore` file.
```

Comment:
```suggestion
From now on, you simply run `git ignore` to launch the interactive CLI, anytime you want to setup or change a `.gitignore` file.
```

The cool thing with git is, we can actually run this as `git ignore`. Would love to keep this as is.